compose/mpp/demo/src/uikitMain/kotlin/bugs/RedundantRecompositionOnStart.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bugs
+
+import androidx.compose.material.Text
+import androidx.compose.mpp.demo.Screen
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalDensity
+
+// https://github.com/JetBrains/compose-multiplatform/issues/3778
+val StartRecompositionCheck = Screen.Example("Start Recomposition Check") {
+    Text("See logs. Density should be printed only once")
```

Comment:
Let's just add global var and `Text("Repompositions: $count")`

Reply:
done